tawk_to.module

```diff
@@ -2,47 +2,30 @@
 
 /**
  * @file
- * @package   tawk.to module for Drupal
+ * @package   tawk.to module for Backdrop CMS
  * @copyright (C) 2021 tawk.to
  * @license   GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
  */
 
-
-define('TAWKTO_WIDGET_PAGE_ID', 'tawkto_widget_page_id');
-define('TAWKTO_WIDGET_WIDGET_ID', 'tawkto_widget_widget_id');
-define('TAWKTO_WIDGET_OPTIONS', 'tawkto_widget_options');
-define('TAWKTO_WIDGET_USER_ID', 'tawkto_widget_user_id');
```

Comment:
I do think we retain these constant variables and just change the value of them.
Ex.
`define('TAWKTO_WIDGET_PAGE_ID', 'tawkto_widget_page_id');` becomes `define('TAWKTO_WIDGET_PAGE_ID', 'page_id');`

This is so we can avoid repeating the hardcoded values below.

Reply:
@alvinjohnsonso I'm not quite sure of the benefits of doing this.  The constants will be longer than the strings they're replacing.  Please can you explain why this is better practice?

Reply:
Since the strings are repeating hard-coded values. I think it would be better to have those in a constant variable (even if it's a longer constant variable name). It'll be cleaner and if ever there's a time that we'll need to update the value of those, they're all just be in one place.

Reply:
Thanks for the explanation.  Done